docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -4,11 +4,11 @@
 [id="hermetic-builds_{context}"]
 = Enabling hermetic builds
 
-Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This enhanced dependency visibility allows the build system to capture and produce a more precise record of the build's origin, dependencies, and modifications than it would otherwise be possible.
+A hermetic build is a controlled build process in which a build system is aware of all required resources and dependencies. With this enhanced visibility, the build system can capture and produce a more precise record of the build's origin, dependencies, and modifications.
 
-In {ProductName}, hermetic builds are achieved by blocking network access to the build process. Therefore, you must pre-fetch all dependencies before the build can start.
+In {ProductName}, you can run a hermetic build by blocking network access to the build process. This means that you must fetch all dependencies _before_ the build can start.
 
-IMPORTANT: Hermetic builds disable network access, so the builds that require dependencies outside its Git repository could fail.
+IMPORTANT: Hermetic builds disable network access, so builds that have dependencies outside of a Git repository could fail.
```

Comment:
IIUC, this should probably be something like `outside of their own Git repository` (dependencies *within* their own repo would be fine, as they would definitely be grabbed during pre-fetch)